include/circt/Dialect/Seq/SeqAttributes.td

```diff
@@ -30,4 +30,16 @@ def WUWAttr : I32EnumAttr<"WUW", "Write-Under-Write Behavior",
 
 }
 
+/// An attribute holding information about memory initialization.
+def FirMemInitAttr : AttrDef<SeqDialect, "FirMemInit"> {
```

Comment:
Can you explain a bit about how the `firmem.init` is supposed to be used ?

Reply:
Oh yeah great point! I modeled this after the FIRRTL-side attribute, but forgot to migrate over the nice description there. Thanks for pointing this out :+1:

Reply:
Done